src/mythiliWorkouts/maxSumArray.java

```diff
@@ -0,0 +1,16 @@
+class Solution {
+    public int maxSubArray(int[] nums) {
+        int maxVal = -9999999;
```

Comment:
nit: We can also use `Integer.MIN_VALUE` -- a better practice in production and interviews.

Reply:
Got it!